packages/database/src/schema.prisma

```diff
@@ -173,3 +180,14 @@ model NsipProject {
   easting  Int?
   northing Int?
 }
+
+model FullAddress {
+  id               String           @id @default(dbgenerated("newid()")) @db.UniqueIdentifier
+  addressLine1     String
+  addressLine2     String
+  townCity         String
+  county           String?
+  country          String
+  postcode         String
+  ApplicantDetails ContactDetails[] @relation("ApplicantAddress")
```

Comment:
Again this needs to be more generic to just the contactDetails relation rather than saying it is for Applicant because it will be for agent and potentially more in future if the form changes. So just ContactDetails is fine